File: implementations/julia/src/RCLCore.jl
module RCLCore
include("rcl/Lexer.jl")
include("rcl/Parser.jl")
include("rcl/Converters.jl")

parse(s::String) = parse_rcl(s)
to_object(input) = to_object_doc(input isa String ? parse_rcl(input) : input)

end

File: implementations/php/src/RCLCore.php
<?php
require_once __DIR__ . '/RCLAst.php';

final class RCLCore {
  public static function parse(string $text): array { return RCLAst::parse($text); }
  public static function toObject(string $text): array { return RCLAst::project(self::parse($text)); }
}
